### PR TITLE
Chore/restore package lock

### PR DESCRIPTION
**Pull request by kpchoy**
**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

File: app/package.json [hunk 1]
```diff
@@ -31,7 +31,6 @@
     "@types/jest": "^26.0.23",
     "@types/react": "^18.0.21",
     "@types/react-native": "^0.70.6",
-    "@types/react-native-auth0": "2.14.1",
```
Comment (mhuynh5757) [inline app/package.json:34]: Do we need this package?

**Review by mhuynh5757 — CHANGES_REQUESTED** [review]

File: app/package.json (hunk 1 above)
Comment (kpchoy) [reply app/package.json:34]: I think before I may have added it to extract a class, but I removed that code. The compiler doesn't complain right now.

Comment (kpchoy) [reply app/package.json:34]: I don't think we need it will add back later if needed.

Comment (mhuynh5757) [reply app/package.json:34]: Sounds good, removing packages always feels good haha.

**Review by charliesantos — APPROVED** [review]
👍

**Pull request merged**